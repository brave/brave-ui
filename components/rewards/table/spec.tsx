/* global jest, expect, describe, it, afterEach */
import * as React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import Table from './index'

describe('Table tests', () => {
  const baseComponent = (props?: object) => <Table id='list' {...props} />

  describe('basic tests', () => {
    it('matches the snapshot', () => {
      const component = baseComponent()
      const tree = create(component).toJSON()
      expect(tree).toMatchSnapshot()
    })

    it('renders the component', () => {
      const wrapper = shallow(baseComponent())
      const assertion = wrapper.find('#list').length
      expect(assertion).toBe(1)
    })
  })
})
