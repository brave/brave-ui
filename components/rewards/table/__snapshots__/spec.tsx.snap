// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table tests basic tests matches the snapshot 1`] = `
.c0 {
  text-align: center;
  padding: 30px 0;
  color: #999ea2;
  font-size: 14px;
}

<div
  className=""
  id="list"
>
  <div
    className="c0"
  />
</div>
`;
